main/utils/errors.js

```diff
@@ -33,7 +33,7 @@ const getSentryIssue = async (eventId, tries = 0) => {
 
     return body;
   } catch (error) {
-    showError(error);
```

Comment:
Add a code comment about this, so we don't accidentally change it back in the future.